roles/telemetry_logging/tasks/main.yml

```
@@ -0,0 +1,5 @@
+---
+- when: journal_list is defined
+  name: "Verify journal entries - {{ journal_polar_id }}"
+  include_tasks: "journal_tests.yml"
+  loop: "{{ journal_list }}"
```

```suggestion
- name: "Verify journal entries - {{ journal_polar_id }}"
  when: journal_list is defined
  ansible.builtin.include_tasks: "journal_tests.yml"
  loop: "{{ journal_list }}"
```

key-order[task]: You can improve the task key order to: name, when, include_tasks, loop
roles/telemetry_logging/tasks/main.yml:2 Task/Handler: Verify journal entries - {{ journal_polar_id }}

https://ansible-lint.readthedocs.io/rules/key-order/

fqcn[action-core]: Use FQCN for builtin module actions (include_tasks).
roles/telemetry_logging/tasks/main.yml:2 Use ansible.builtin.include_tasks or ansible.legacy.include_tasks instead.

https://ansible-lint.readthedocs.io/rules/fqcn/

---

done